samples/batch/python/python-client/main.py

```diff
@@ -25,118 +23,180 @@
 LOCALE = "en-US"
 RECORDINGS_BLOB_URI = "<Your SAS Uri to the recording>"
 
-# Set subscription information when doing transcription with custom models
-ADAPTED_ACOUSTIC_ID = None  # guid of a custom acoustic model
-ADAPTED_LANGUAGE_ID = None  # guid of a custom language model
+# Provide the uri of a container with audio files for transcribing all of them with a single request
+RECORDINGS_CONTAINER_URI = "<Your SAS Uri to a container of audio files>"
+
+# Set model information when doing transcription with custom models
+MODEL_REFERENCE = None  # guid of a custom model
+
+
+def transcribe_from_single_blob(uri, properties):
+    """
+    Transcribe a single audio file located at `uri` using the settings specified in `properties`
+    using the base model for the specified locale.
+    """
+    transcription_definition = cris_client.Transcription(
```

Comment:
is it "transcription definition" or already "transcription response"?

Reply:
Here, it is really the definition (hasn't been sent yet)